homeassistant/components/matter/config_flow.py

```diff
@@ -34,6 +34,7 @@
 ADDON_SETUP_TIMEOUT = 5
 ADDON_SETUP_TIMEOUT_ROUNDS = 40
 DEFAULT_URL = "ws://localhost:5580/ws"
+DEFAULT_TITLE = ""  # empty string = same name as integration
```

Comment:
We shouldn't use empty string as title. We could set it just to Matter or similar. Hiding the title completely is a frontend concern.

Reply:
I was told this is an "hidden" feature of the frontend. It will show the integration name as Title if you leave the title blank, which is exactly what I want.

Alternative is we set the title to the integration title ourselves in the code.

Reply:
We should not care about how it looks in the frontend in the backend. I suggest setting it to just Matter or leaving it as is.